runtime/mainnet/Cargo.toml

```diff
@@ -39,6 +39,7 @@ frame-support.workspace = true
 frame-system-benchmarking = { workspace = true, optional = true }
 frame-system-rpc-runtime-api.workspace = true
 frame-system.workspace = true
+pallet-assets.workspace = true
```

Comment:
do we need this in the PR scope?

Reply:
No, we don't.
This is extra.
I removed it.